cppcache/src/ExceptionTypes.cpp

```diff
@@ -30,8 +30,8 @@ namespace client {
 void setThreadLocalExceptionMessage(std::string exMsg);
 const std::string& getThreadLocalExceptionMessage();
 
-std::map<GfErrType,
-         std::function<void(std::string, std::string&, GfErrType, std::string)>>
+static std::map<GfErrType, std::function<void(std::string, std::string&,
+                                              GfErrType, std::string)>>
```

Comment:
Can we make this conform to the naming convention. 

Will be nice if some day this could be `constexpr`. 
https://www.youtube.com/watch?v=INn3xa4pMfg

Reply:
Got it!  I'll have some tests run again just to make sure it is still green